lib/index.js

```diff
@@ -74,12 +78,6 @@ Appboy.prototype.loaded = function() {
  */
 
 Appboy.prototype.identify = function(identify) {
-  var userId = identify.userId();
-  // Appboy doesn't want you to call changeUser unless a userId is present because they do automatic anonymous user tracking on their end 
-  // https://www.appboy.com/documentation/Web/#setting-user-ids
-  if (!userId) return;
-  window.appboy.changeUser(userId);
```

Comment:
@hankim813 I reached out to them to verify, but it is currently being called in `initialize()`: https://github.com/segment-integrations/analytics.js-integration-appboy/blob/master/lib/index.js#L51